tgui/packages/fulpui-patches/interfaces/AntagInfoBloodsucker.tsx

```diff
@@ -105,22 +152,114 @@ const BloodsuckerIntro = () => {
+          />
+        </Box>
+      </Section>
+    );
+  }
+
   return (
-    <Window width={620} height={580}>
-      <Window.Content>
-        <Tabs>
-          <Tabs.Tab
-            icon="list"
-            lineHeight="23px"
-            selected={tab === 1}
-            onClick={() => setTab(1)}>
-            Introduction
-          </Tabs.Tab>
-        </Tabs>
-        {tab === 1 && <BloodsuckerIntro />}
-      </Window.Content>
-    </Window>
+    <Stack vertical fill>
+      <Stack.Item minHeight="20rem">
+        <Section scrollable fill>
+          <Stack vertical>
+            <Stack.Item>
+              {clan.map((ClanInfo) => (
+                <>
+                  <Box
+                    as="img"
+                    height="20rem"
+                    opacity={0.25}
+                    src={resolveAsset(`bloodsucker.${ClanInfo.clan_icon}.png`)}
+                    style={{
+                      '-ms-interpolation-mode': 'nearest-neighbor',
+                      'position': 'absolute',
+                    }}
+                  />
+                  <Stack.Item fontSize="20px" textAlign="center">
+                    You are part of the {ClanInfo.clan_name}
+                  </Stack.Item>
+                  <Stack.Item fontSize="16px">
+                    {ClanInfo.clan_description}
+                  </Stack.Item>
+                </>
+              ))}
+            </Stack.Item>
+          </Stack>
+        </Section>
+        <PowerSection />
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+const PowerSection = (props: any, context: any) => {
+  const { act, data } = useBackend<BloodsuckerInformation>(context);
+  const { power } = data;
+  const [selectedPower, setSelectedPower] = useSharedState(
+    context,
+    'power',
+    power[1]
+  );
+  return (
+    <Section
+      fill
+      scrollable={!!power}
+      title="Powers"
+      buttons={
+        <Button
+          icon="info"
+          tooltipPosition="left"
+          tooltip={'Select a Power to explain.'}
+        />
+      }>
+      <Stack>
+        <Stack.Item grow>
+          <Dropdown
+            displayText={selectedPower.power_name}
+            selected={selectedPower.power_name}
+            width="100%"
+            options={power.map((powers) => powers.power_name)}
+            onSelected={(powerName: string) =>
+              setSelectedPower(
+                power.find((p) => p.power_name === powerName) || power[0]
+              )
+            }
+          />
+          {!!selectedPower && (
+            <Box
+              position="absolute"
+              height="12rem"
+              as="img"
+              src={resolveAsset(`bloodsucker.${selectedPower.power_icon}.png`)}
+            />
+          )}
+          <Divider Vertical />
+        </Stack.Item>
+        <Stack.Divider />
+        <Stack.Item scrollable grow={1} fontSize="16px">
+          {!!selectedPower && selectedPower.power_explanation}
```

Comment:
Boolean true isn't needed. Expression can be boolean true or truthy
```suggestion
          {selectedPower && selectedPower.power_explanation}
```